docs/tests/conditionals.rst

```diff
@@ -87,16 +79,13 @@ patterns. The regex value must FULLY match the key associated with it:
 .. code:: yaml
 
     basic_regex_test:
+        # In this example we see keys accepting regex patterns.
         only_if:
+            # This test will run if the user is a lowercase [a-z] word.
             "{{user}}": ['^[a-z]+$']
+            # This test will only run if the 'sys_os' is linux.
+            # Pavilion anchors the values so linux will only match linux.
```

Comment:
What does it mean to 'anchor' a value?
`Keys must be fully matched by the regex.`